/**
 * Versions related methods within a service.
 * This service will be used to get the version of infrastructure components from outside the cluster.
 */
syntax = "proto3";

package armonik.api.grpc.v1.versions;

import "versions_common.proto";

option csharp_namespace = "ArmoniK.Api.gRPC.V1.Versions";
option java_package = "fr.aneo.armonik.api.grpc.v1.versions";

/**
 * Service for handling versions.
 */
service Versions {
  /**
   * Get all versions.
   */
  rpc ListVersions(ListVersionsRequest) returns (ListVersionsResponse) {}
}
